controls/4_1_configure_system_accounting_auditd.rb

```diff
@@ -217,9 +217,7 @@
   only_if { cis_level == 2 }
 
   describe file('/etc/audit/audit.rules') do
-    its(:content) { should match(%r{^-w /var/log/faillog -p wa -k logins$}) }
```

Comment:
can you comment these out rather than removing them, and add a comment as to why we have disabled these checks

Reply:
sure